### ISSUE TITLE
netxml output does not convert "&" to xml &amp;

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#753]
**Reported by lord carlos on 6 Jun 2010 14:38 UTC**

If airodump-ng writes to kismet netxml output, it should convert "&" from <manuf> field into &amp;

----

```
...<BSSID>00:11:22:33:44:55</BSSID>
		<manuf>CC&C Technologies, Inc.</manuf>
		<channel>7</channel>
		<freqmhz>2442 2</freqmhz>
		<maxseenrate>54000</maxseenrate>...
```

The "CC&C" from <manuf> should be something like this: CC&amp;C

----
I do not know if the converting also fails in the essid field.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 6 Jun 2010 14:38 UTC**

The conversation was done for the ESSID but not for the manufacturer.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 6 Jun 2010 14:38 UTC**

Fixed in aircrack-ng/aircrack-ng@19b0dccd3bb8086b8f42ee86d057b97d0bd6bf34
